index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+        <div>
+          <button id="Up Vote">-</button>
+          <input type = "number" value="0"></input>
+          <button id="Down Vote">-</button>
+        </div>
+        <script scr="script.js"></script>
+</body>
+</html>
+
```

Comment:
So this is the way to do buttons if you weren't using a framework like React. As you are using React, you don't need to do this as you can implement the feature using React!